src/utils.jl

```diff
@@ -19,6 +19,65 @@ cholesky_factor(X::Cholesky) = X.U
 cholesky_factor(X::UpperTriangular) = X
 cholesky_factor(X::LowerTriangular) = X
 
+# TODO: Add `check` as an argument?
```

Comment:
Should we?

Reply:
I think this is the last remaining question @devmotion . I'm thinking "let's not, until we start using it"?